ramp-engine/ramp_engine/dispatcher.py

```diff
@@ -198,14 +198,20 @@ def collect_result(self, session):
                 self._processing_worker_queue.put_nowait(
                     (worker, (submission_id, submission_name)))
                 time.sleep(0)
+            elif worker.status == 'reset':
+                self._reset_single_submission(
+                    session, submission_id
+                )
+                logging.info(f'Submission: {submission_id} has been reset due '
+                             'to spot instance being terminated by AWS.')
```

Comment:
Maybe only log this when
```
isinstance(worker, AWSWorker)
```
and otherwise print a more generic error about the submission training being interrupted and that it will be retried?